Tests/MapboxDirectionsTests/V5Tests.swift

```diff
@@ -63,7 +63,7 @@ class V5Tests: XCTestCase {
         }
         XCTAssertNotNil(task)
         
-        waitForExpectations(timeout: 10) { (error) in
+        waitForExpectations(timeout: 15) { (error) in
```

Comment:
Decoding now takes a lot more time... This especially affects `geoJSON` formatted responses (see `testGeoJSON` time increased from [~2.7s](https://app.circleci.com/pipelines/github/mapbox/mapbox-directions-swift/1478/workflows/2ce59a0c-b58d-462f-a3ca-1e4e0f030a44/jobs/5838) to [~12s](https://app.circleci.com/pipelines/github/mapbox/mapbox-directions-swift/1480/workflows/1835e71e-dee8-4796-b998-031fdbf2b0ba/jobs/5850)). I have no ideas on how to improve performance on this without re-designing the entire JSON roundtripping mechanics, starting from Turf. On my local machine, this test time is increased from `1s` to `5s`.

Reply:
Since it’s extremely unlikely for a GeoJSON-formatted geometry in a Directions API response to contain any foreign members or for these foreign members to have any significance, perhaps Turf should look for an optional `userInfo` key indicating that foreign members should be ignored when encoding or decoding. Turf could recursively pass down this userInfo key. This would be a generally useful feature.

In the meantime, I’m not terribly concerned about a performance hit limited to GeoJSON-formatted geometries, since the navigation SDK’s NavigationRouteOptions defaults to Polyline encoding, partly for performance reasons.

Reply:
> Since it’s extremely unlikely for a GeoJSON-formatted geometry in a Directions API response to contain any foreign members or for these foreign members to have any significance, perhaps Turf should look for an optional `userInfo` key indicating that foreign members should be ignored when encoding or decoding. Turf could recursively pass down this userInfo key. This would be a generally useful feature.

Tracking in mapbox/turf-swift#186.